action.yml

```diff
@@ -108,8 +115,10 @@ runs:
         if: inputs.skip-go-get != 'true'
         shell: bash
         working-directory: ${{ runner.temp }}
+        env:
+          MODULE_SUFFIX: ${{ steps.major_version.outputs.major_version > 1 && format('/v{0}', steps.major_version.outputs.major_version) || '' }}
```

Comment:
what does the {0} stand for here?

Reply:
The zeroth argument to the `format()` function - `steps.major_version.outputs.major_version`